### PR TITLE
Add example for slash commands in cogs.

### PR DESCRIPTION
**Pull request by Sengolda**
## Summary

This pull request adds an example for slash commands in cogs.

## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new
- [ ] This PR is a breaking change
- [x] This PR is **not** a code change (e.g. documentation, README, typehinting, ...)

### CODE REVIEW

**Review by Lulalaby — DISMISSED** [review]

**Comment by Dorukyum** [comment]
It could be better if you put some comments to explain the code

**Comment by Lulalaby** [comment]
> 
> 
> It could be better if you put some comments to explain the code

true

**Comment by Sengolda** [comment]
> It could be better if you put some comments to explain the code

Every part? or just the parts where we are making slash commands?

**Comment by Lulalaby** [comment]
> > It could be better if you put some comments to explain the code
> 
> Every part? or just the parts where we are making slash commands?

Document it so newbies can understand what's happening there.

File: examples/app_commands/slash_cog.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+class Example(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+ 
+    @slash_command(guild_ids=[...]) # Create a slash command for the supplied guilds.
+    async def hello(self, ctx):
+        await ctx.respond("Hi, this is a slash command from a cog!")
+    
+
+    @slash_command() # Not passing in guild_ids creates a global slash command (might take an hour to register).
```
Comment (Lulalaby) [inline examples/app_commands/slash_cog.py:13]: Tell the users that this is a global command

File: examples/app_commands/slash_cog.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+from discord.ext import commands
+from discord.commands import slash_command # Importing the decorator that makes slash commands.
+
+class Example(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+ 
+    @slash_command(guild_ids=[...]) # Create a slash command for the supplied guilds.
```
Comment (Lulalaby) [inline examples/app_commands/slash_cog.py:8]: Tell the users that this is a guild command

**Review by Lulalaby — CHANGES_REQUESTED** [review]

File: examples/app_commands/slash_cog.py (hunk 1 above)
Comment (Sengolda) [reply examples/app_commands/slash_cog.py:13]: `creates a global slash command` it's already there wdym

Comment (Lulalaby) [reply examples/app_commands/slash_cog.py:13]: oops

**Review by Lulalaby — APPROVED** [review]

**Review by Dorukyum — APPROVED** [review]

**Pull request merged**